packages/lesswrong/components/forumEvents/votingPortal/VotingPortalPage.tsx

```diff
@@ -793,7 +793,7 @@ const ThankYouScreen = ({
   const updateCurrentUser = useUpdateCurrentUser();
   const {captureEvent} = useTracking();
   const [addFlair, setAddFlair] = useState(
-    currentUser.givingSeason2024VotedFlair ?? false,
+    false,
```

Comment:
Maybe we want to keep this one? The voting portal isn't accessible anyway, and we prob want to just update this value for next year (assuming we do another donation election)